### PR TITLE
Add svcutilcore target, prefercliruntime and json files.

### PR DESCRIPTION
**Pull request by huanwu**
Add missing files for svcutilcore.pkgproj. 

I am not sure if I can add these files since since there's "build/" in .gitignore file.
@zhenlan @mconnew @StephenBonikowsky

### CODE REVIEW

File: src/svcutilcore/pkg/files/svcutilcore.targets [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+    <_SerializationAssemblyName>$(AssemblyName).XmlSerializers</_SerializationAssemblyName>
+    <_SerializerDllIntermediateFolder>$(IntermediateOutputPath)$(_SerializationAssemblyName).dll</_SerializerDllIntermediateFolder>
+    <_SerializerPdbIntermediateFolder>$(IntermediateOutputPath)$(_SerializationAssemblyName).pdb</_SerializerPdbIntermediateFolder>
+    <_SerializerCsIntermediateFolder>$(IntermediateOutputPath)$(_SerializationAssemblyName).cs</_SerializerCsIntermediateFolder>
+    <_SvcUtilWarningText>Warning : Fail to generate the serializer for $(AssemblyName)$(TargetExt).</_SvcUtilWarningText>
+    <_SerializationAssemblyDisabledWarnings>$(NoWarn);219;162;$(SerializationAssemblyDisabledWarnings)</_SerializationAssemblyDisabledWarnings>
+  </PropertyGroup>
+
+  <Target Name="SvcUtilGenerateSerializationAssembly" AfterTargets="Build">
+	  <Message Text="Path3 = $(IntermediateOutputPath)" Importance="high" />
```
Comment (zhenlan) [inline src/svcutilcore/pkg/files/svcutilcore.targets]: NIT: please fix the indentation.

BTW, why Path**3**?

File: src/svcutilcore/pkg/files/svcutilcore.targets [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+    <Warning Condition="Exists('$(_SerializerCsIntermediateFolder)') != 'true'" Text="$(_SvcUtilWarningText)" />
+    <Csc Condition="Exists('$(_SerializerCsIntermediateFolder)') == 'true'" ContinueOnError="true" OutputAssembly="$(_SerializerDllIntermediateFolder)" References="@(ReferencePath);@(IntermediateAssembly)" EmitDebugInformation="$(DebugSymbols)" Sources="$(_SerializerCsIntermediateFolder)" TargetType="Library" ToolExe="$(CscToolExe)" ToolPath="$(CscToolPath)" DisabledWarnings="$(_SerializationAssemblyDisabledWarnings)" />
+    <Warning Condition="Exists('$(_SerializerDllIntermediateFolder)') != 'true' And Exists('$(_SerializerCsIntermediateFolder)') == 'true'" Text="$(_SvcUtilWarningText)" />
+    <Copy Condition="Exists('$(_SerializerDllIntermediateFolder)') == 'true'" SourceFiles="$(_SerializerDllIntermediateFolder)" DestinationFolder="$(OutputPath)" />
+  </Target>
+  
+  <Target Name="SvcUtilCleanSerializationAssembly" AfterTargets="CoreClean">
+    <Message Text="Cleaning serialization files..." Importance="normal" />
```
Comment (zhenlan) [inline src/svcutilcore/pkg/files/svcutilcore.targets]: Cleaning up serialization assemblies...

File: src/svcutilcore/pkg/files/dotnet-svcutilcore.runtimeconfig.json [hunk 3]
```diff
@@ -0,0 +1,9 @@
+{
+  "runtimeOptions": {
+    "tfm": "netcoreapp2.0",
```
Comment (zhenlan) [inline src/svcutilcore/pkg/files/dotnet-svcutilcore.runtimeconfig.json]: We require it to be "netcoreapp2.1", don't we?

File: src/svcutilcore/pkg/files/svcutilcore.targets (hunk 1 above)
Comment (huanwu) [reply src/svcutilcore/pkg/files/svcutilcore.targets]: That's used for debug purpose. Will delete it.

File: src/svcutilcore/pkg/files/dotnet-svcutilcore.runtimeconfig.json [hunk 4]
```diff
@@ -0,0 +1,9 @@
+{
+  "runtimeOptions": {
+    "tfm": "netcoreapp2.1",
+    "framework": {
+      "name": "Microsoft.NETCore.App",
+      "version": "2.0.0"
```
Comment (zhenlan) [inline src/svcutilcore/pkg/files/dotnet-svcutilcore.runtimeconfig.json]: I am actually not very sure if 2.0.0 MS.NC.App will work with tfm netcoreapp2.1. We probably should remove the framework section altogether or specify a particular version, for example, the version .NET Core just released for 2.1 preview 1.

Anyway, if this passes CI, let's still get this in and do a test run on VSTS. I'd like to unblock nightly run first.

**Pull request merged**